org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -46,7 +89,27 @@ public class MRTKRayInteractor :
         /// Used to check if the parent controller is tracked or not
         /// Hopefully this becomes part of the base Unity XRI API.
         /// </summary>
-        private bool IsTracked => xrController.currentControllerState.inputTrackingState.HasPositionAndRotation();
+        private bool IsTracked
+        {
+            get
+            {
+                #pragma warning disable CS0618 // Type or member is obsolete
+                if (xrController == null) //If no XRController is associated with this interactor then try to get the TrackedPoseDriver component instead
+                {
+                    if (TrackedPoseDriver == null) //If the interactor does not have a TrackedPoseDriver component then it is not tracked
+                    {
+                        return false;
+                    }
+
+                    //If this interactor has a TrackedPoseDriver then use it to check if this interactor is tracked
+                    ((InputTrackingState)TrackedPoseDriver.trackingStateInput.action.ReadValue<int>()).HasPositionAndRotation();
```

Comment:
The pose of this interactor doesn't come for a `TrackedPoseDriver` it's coming from `AimPoseSource`.   Weird I know.   So, we can probably use `AimPoseSource` to determine track state.  See my draft PR https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/745/files

But I suppose using `TrackedPoseDriver` aligns more with what XRI2 version did...

So I'm not sure

Reply:
Let's go with the TrackedPoseDriver as I believe that's Unity recommendation to track position and orientation in the context of XRController removal.